src/main/java/com/epam/eco/commons/kafka/consumer/advanced/AdvancedConsumer.java

```diff
@@ -162,6 +180,16 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
                     if (!records.isEmpty()) {
                         submitNewHandlerTask(records);
                     }
+                } catch (AuthenticationException | AuthorizationException ae) {
```

Comment:
Let's leave  AuthorizationException only, since AuthenticationException seems to be unrecoverable